name.abuchen.portfolio/src/name/abuchen/portfolio/money/Values.java

```diff
@@ -64,9 +64,9 @@ public String formatNonZero(Money amount, String skipCurrencyCode)
 
     public static final class QuoteValues extends Values<Long>
     {
-        private static final String QUOTE_PATTERN = "#,##0.00######"; //$NON-NLS-1$
+        static final String QUOTE_PATTERN = "#,##0.00######"; //$NON-NLS-1$
 
-        private static final ThreadLocal<DecimalFormat> QUOTE_FORMAT = ThreadLocal // NOSONAR
+        static final ThreadLocal<DecimalFormat> QUOTE_FORMAT = ThreadLocal // NOSONAR
```

Comment:
Do I understand it correct, that this change is made to facility testing? Because you need to change the locale, but the DecimalFormat in the ThreadLocal also needs to change? If so, then I think an explicit method is also okay. If we annotate it with ```@VisibleForTesting``` then the IDE even provides hints in case we accidentally want to use it.

Reply:
I've made the change to be able to change the Locale during testing in order to replace the instance of DecimalFormat in QUOTE_FORMAT after it has changed. Concerning a method: I've implemented one in ValuesBuilder.initQuoteValuesDecimalFormat which is why I've changed the visibility of these two members in the first place. So I'm not sure if another method in non-test-code is necessary.